lib/gollum/views/page.rb

```diff
@@ -184,6 +192,42 @@ def content_without_page_header(content)
         # .inner_html will cause href escaping on UTF-8
         doc.css("div#gollum-root").children.to_xml(@@to_xml)
       end
+
+      def table(data)
+        result = "<table>\n"
+        case data
+          when Hash
```

Comment:
The alternative is to just print `Array.to_s`. That's alright in principle, but then I think we also shouldn't iterative over `Hash`es but just print `to_s` for every value, so basically only have a "first order" table. My preference would be to do what github does, but I don't actually know how they handle `Array`s and `Hash`es in front matter. :)